src/containers/quiz/quiz-question/Question.tsx

```diff
@@ -0,0 +1,158 @@
+import { ChangeEventHandler, FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box, Typography, FormGroup, RadioGroup, SxProps } from '@mui/material'
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+import AppCard from '~/components/app-card/AppCard'
+import CheckIcon from '@mui/icons-material/Check'
+import CloseIcon from '@mui/icons-material/Close'
+
+import { Question } from '~/types'
+import { styles } from './Question.styles'
+import { isCorrectAnswer } from './Question.constants'
+import Answer from '../question-answer/Answer'
+import { spliceSx } from '~/utils/helper-functions'
+
+interface QuizQuestionProps {
+  question: Question
+  index: number
+  value: string | null | string[]
+  showCorrectAnswers?: boolean
+  showPoints?: boolean
+  showAnswersCorrectness?: boolean
+  isEditable?: boolean
+  useAppCard?: boolean
```

Comment:
Refactor this name, because in the code later it can be mixed up with hook and it becomes confusing

Reply:
Thanks, changed it for `appCardWrapper`